### DISCUSSION TITLE
Getting photo_url from a user object

### DISCUSSION THREAD

**Discussion by frenme** [revenkroz/telegram-web-app-bot-example#22] [Q&A]
Hi!
Please can you tell how you get 'photo_url' from the object 'window.Telegram.WebApp.initDataUnsafe' ? 
The documentation says that 'photo_url' can only be requested if you run a bot from attachments.

**Comment by Kavinsky1337** [comment]
Hello, u cant get photo_url in tg bot with buttons, u want get with attched menu(for example: tg wallet)

If u want user photo, u cant download it with tg bot, if user /start bot. Add photo to database and using it

**Reply by Kavinsky1337** [reply]
***I can dowload photo

**Reply by fukemy** [reply]
I dont know whats u mean
